modules/tinymce/src/plugins/table/test/ts/browser/ModifyColumnsTableResizeTest.ts

```diff
@@ -378,6 +448,29 @@ describe('browser.tinymce.plugins.table.ModifyColumnsTableResizeTest', () => {
           mceTableDeleteCol: 0
         });
       });
+
+      it('TINY-6711: should resize table by single column width with a colspan', () => {
```

Comment:
I'm wondering whether the new column should have the width of colspan td if that's where the selection is? I will need to have a play to see

Reply:
As far as I'm aware it's not meant to. You'll see stable will make it the width of a single column not multiple columns. There's also a number of tests that assert that.

Reply:
You'll also see the copy column logic does the same thing, where it works based on a single column when a colspan exists.

Reply:
We could certainly change that behaviour, but it's not something we should be looking to do given the time constraints. If we did do that though, then there's likely to be an expectation multiple columns would be inserted.